00_introduction/21_architecture_root_server.txt

```diff
@@ -166,17 +163,16 @@ Node. It also contains an init script:
 
 ----
 
-/etc/init.d/rudder-agent 
+service rudder-agent 
```

Comment:
Nope, this is a reference to the init script, it can't be replaced by a command :)